### PR TITLE
Add CSV support

### PR DESCRIPTION
**Pull request by ppai-plivo**
Running `fetch.js` now also generates `mcc-mnc-list.csv` file.

```sh
$ ls -lh mcc-mnc-list.*
-rw-r--r--  1 user  staff   283K Apr  8 13:36 mcc-mnc-list.csv
-rw-r--r--  1 user  staff   763K Apr  8 13:23 mcc-mnc-list.json
```

Closes: #15

### CODE REVIEW

File: fetch.js [hunk 1]
```diff
@@ -4,6 +4,7 @@ const fs = require('fs');
 const path = require('path');
 const jsdom = require('jsdom');
 const { JSDOM } = jsdom;
+const csvstringify = require('csv-stringify/lib/sync')
```
Comment (ppai-plivo) [inline fetch.js:7]: As documented here: https://csv.js.org/stringify/api/#sync-api

**Comment by ppai-plivo** [comment]
@pbakondy please let me know if this is ok.

**Comment by pbakondy** [comment]
@ppai-plivo please include only the code changes without the generated files in your patch

**Comment by ppai-plivo** [comment]
> @ppai-plivo please include only the code changes without the generated files in your patch

Done 👍 

I hope that next time you run `fetch.js` and update the list, `mcc-mnc-list.csv` is checked in and makes it to the repo. I need it to be downloadable from github, similar to `mcc-mnc-list.json` file.

**Comment by ppai-plivo** [comment]
@pbakondy Can you please take a look again ? Thanks.

**Comment by ppai-plivo** [comment]
Bump!

**Pull request closed without merging**